### DISCUSSION TITLE
Would you build a discussion forum using Gatsby?

### DISCUSSION THREAD

**Discussion by cesarvarela** [gatsbyjs/gatsby#31989] [Help]
Hi, I'm wondering how a forum would work using Gatsby, I successfully used it to build a few websites, but this seems slightly different. Has someone done it? Are there any working examples? Thanks!

**Comment by herecydev** [comment]
It's probably a bad idea. The nature of a forum involves rapidly changing pages (both the "home page" and a "thread" are very fluid). Even if you were regenerating the pages rapidly, it would cause the client's to reload (so they wouldn't get a SPA experience).

If it were me, I would look to do this in nextjs or something similar, it's just a better fit for that level of dynamic data.
